src/engine.py

```
@@ -5816,7 +5814,27 @@ async def on_provider_event(self, event: Dict[str, Any]):
+                        if provider_name and provider_name in self.config.providers:
+                            provider_cfg = self.config.providers.get(provider_name, {})
+                            provider_delay = provider_cfg.get('farewell_hangup_delay_sec') if isinstance(provider_cfg, dict) else getattr(provider_cfg, 'farewell_hangup_delay_sec', None)
+                            if provider_delay is not None:
+                                hangup_delay = provider_delay
+                                logger.debug(
+                                    "Using provider-specific farewell delay",
+                                    call_id=call_id,
+                                    provider=provider_name,
+                                    delay=hangup_delay
+                                )
+                except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)